internal/tests/vdc/vdc_group_datasource_test.go

```diff
@@ -0,0 +1,35 @@
+package vdc
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	tests "github.com/orange-cloudavenue/terraform-provider-cloudavenue/internal/tests/common"
+)
+
+//go:generate go run github.com/FrangipaneTeam/tf-doc-extractor@latest -filename $GOFILE -example-dir ../../../examples -test
+const testAccVDCGroupDataSourceConfig = `
+data "cloudavenue_vdc_group" "example" {
+	name = "MyVDCGroup"
+}
+`
+
+const dataSourceName = "data.cloudavenue_vdc_group.example"
+
+func TestAccCloudavenueVdcGroupDataSource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { tests.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: tests.TestAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
```

Comment:
Put your const in func. 

```suggestion
func TestAccCloudavenueVdcGroupDataSource(t *testing.T) {
	const dataSourceName = "data.cloudavenue_vdc_group.example"
	resource.Test(t, resource.TestCase{
		PreCheck:                 func() { tests.TestAccPreCheck(t) },
		ProtoV6ProviderFactories: tests.TestAccProtoV6ProviderFactories,
		Steps: []resource.TestStep{
```